packages/slate-editor/src/components/FancyScrollbars.tsx

```diff
@@ -84,17 +83,24 @@ export class FancyScrollbars extends Component<Props> {
 }
 
 function getScrollParent(element: HTMLElement): HTMLElement | null {
-    if (element == null) {
-        return null;
-    }
+    let current = element;
 
-    if (element.scrollHeight > element.clientHeight) {
-        return element;
-    }
+    do {
+        const parent = current.parentElement;
 
-    if (element.parentElement) {
-        return getScrollParent(element.parentElement);
-    }
+        if (parent === null) return null;
+
+        if (parent.scrollHeight > parent.clientHeight && isScrollable(parent)) {
+            return parent;
+        }
+
+        current = parent;
+    } while (current);
```

Comment:
There was a bug here.

For me, it was selecting the menu item itself as its scroll parent (i.e. it was a bug in `getScrollParent()`). Checking `overflowY` and excluding the current element is the fix.

Additionally, I've rewritten the function to a loop, instead of a recursion (yeah, it's a micro-optimization).